prac_05/colour_code.py

```diff
@@ -0,0 +1,12 @@
+
+COLOUR_NAMES = {'ALICE BLUE': '#f0f8ff', 'ANTIQUE_WHITE': '#faebd7', 'ANTIQUE WHITE1': '#ffefdb',
```

Comment:
try having the dict name "colours_to_hexes" and properly have the key names together with no spaces and lowercase.